index.html

```diff
@@ -1,277 +1,334 @@
-    <link rel="stylesheet" href="style.css" />
-  </head>
-  <body>
-    <!--Header part start-->
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Sazzad's Portfolio</title>
+  <link rel="stylesheet" href="style.css" />
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@500&family=Poppins&family=Roboto:wght@700&display=swap"
+    rel="stylesheet" />
+</head>
+
+<body id="body">
+
+  <!--------- header start----------->
+  <main id="main">
     <header>
-      <nav class="navbar">
-        <img src="images/Logo Placeholder.png" alt="logo" />
-        <a class="toggler" target="_self" href="nav-mobile.html">
-          <img src="images/Icon - Menu.png" alt="icon image" />
-        </a>
+      <nav class="navbar-container">
+        <div class="logo-container">
+          <img class="branding-log" src="#" alt="Sazzad" t />
+          <img id="hamImg" src="./icons/Enabled.svg" alt="hamburger menu" />
+        </div>
+        <div class="mobile-menu">
+          <img id="crossImg" src="./icons/cross.png" alt="menu toggler link" />
+          <ul>
+            <li>
+              <a class="mob-items" href="#portfolio" target="_self" rel="noopener">Portfolio</a>
+            </li>
+            <hr>
+            <li>
+              <a class="mob-items" href="#about" target="_self" rel="noopener">About</a>
+            </li>
+            <hr>
+            <li>
+              <a class="mob-items" href="#form" target="_self" rel="noopener">Contact</a>
+            </li>
+            <hr>
+          </ul>
+        </div>
+        <ul class="desk-menu">
+          <li>
+            <a href="index.html" target="_self" rel="noopener">Portfolio</a>
+          </li>
+          <li>
+            <a href="#about" target="_self" rel="noopener">About</a>
+          </li>
+          <li>
+            <a href="#form" target="_self" rel="noopener">Contact</a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/abdus-sattar-a41a26215/" target="_blank" rel="noopener">
+              <img src="icons/Mail.png" alt="message link">
+            </a>
+          </li>
+        </ul>
       </nav>
     </header>
-    <!--Main part start-->
-    <section class="main">
-      <article class="main-article">
-        <h2>Hey There.
-            I’m Sazzad</h2>
-        <h3>I’am a Software Developer</h3>
-        <p>I can help you builds a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-            don’t hestiate to contact me.</p>
-      </article>
-      <div class="social-media">
+    <!-- headline-container start -->
+    <section class="section">
```

Comment:
You have done a great effort implementing the desktop version design. But I would like to mention few things that I observed.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/221349304-715eecc2-a7ca-44ab-a69d-e89fabca380a.png)| ![image](https://user-images.githubusercontent.com/61361037/221349212-5f5e4b00-9e5d-4f86-a660-d5a05dbe3785.png)|

So, I found out that there are two element that would be aligned separately:
1. The text section: Where it is not vertically centered. But before that your section should have acquired the whole screen size, for which you can also use `height: 100vh;` here. I tested it out, it's working. Test it out, But try other things too, if needed. 
After doing background alignment, then align the text in the vertical center.
2. The background Image: The image is hidden above. So you need to push it down. For which you need to add padding to the `.section`. I have tested it out successfully.